lesson10/jquery.js

```diff
@@ -0,0 +1,24 @@
+const express = require("express");
+const bodyParser= require("body-parser");
+const app = express();
+const gm = require("gm");
+const fs = require("fs");
+
+app.listen(3615);
+app.use(express.static('public'));
+app.use(bodyParser.json({limit:'500mb'}));
+
+app.post('/nameCom', (req,res)=>{
+  fs.writeFile('/home/sya/garageScript/lesson10/public/ncom.txt', ` Name: ${req.body.name} Comment: ${req.body.comment}`)
+  res.send("name and comment have been saved ");
+})
+
+
+app.post('/pics', (req,res)=>{
```

Comment:
Try to make your url more readable, like /pics can be /savePicsAsMemes, so are the others. It would be much easier for others

Reply:
made change